ebitenutil/fs_go116.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 The Ebitengine Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+
+//go:build go1.16
+// +build go1.16
```

Comment:
go116?

Reply:
https://github.com/golang/go/blob/master/src/go/build/build.go#L328-L337
I think it's fine as it is